src/fairness/reader/data_reader_db.cpp

```diff
@@ -245,8 +245,7 @@ std::shared_ptr<weighted_tree_node_t> data_reader_db_t::get_sub_banks (
         while (rc == SQLITE_ROW) {
             std::string username = reinterpret_cast<char const *> (
                 sqlite3_column_text (c_assoc, 0));
-            std::string shrs = reinterpret_cast<char const *> (
-                sqlite3_column_text (c_assoc, 1));
+            uint64_t shrs = sqlite3_column_int64 (c_assoc, 1);
```

Comment:
Is there no danger that the shares might grow big enough that the signed/unsigned conversion here becomes a problem?

Reply:
Yeah that seems reasonable!

Reply:
Cool! I'll set MWP here. Thanks for the review